blazor/gantt-chart/resource-multi-taskbar.md

```diff
@@ -0,0 +1,156 @@
+
+                new TaskData() { TaskId = 12, TaskName = "Sign contract",StartDate = new DateTime(2019, 04, 12), Duration = "5",  ParentId = 9, Work=60,Progress=40, TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=3} },Predecessor = "11FS-5" },
+
+                new TaskData() { TaskId = 13,TaskName = "Foundation",StartDate = new DateTime(2022, 04, 04),EndDate = new DateTime(2019,04,28),Work=60,Progress=40,TaskType="FixedDuration" },
+                
+                new TaskData() { TaskId = 14, TaskName = "Excavate for foundations", StartDate = new DateTime(2019, 04, 01),Duration = "2", ParentId = 13, Work=60,Progress=40, TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=4,} } },
+
+                new TaskData() { TaskId = 15, TaskName = "Dig footer",StartDate = new DateTime(2019, 04, 04), Duration = "2", ParentId = 13, Work=60,Progress=40, TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=4} },Predecessor = "14FS + 1" },
+
+                new TaskData() { TaskId = 16, TaskName = "Install plumbing grounds",StartDate = new DateTime(2019, 04, 08), Duration = "2", ParentId = 13, Work=60,Progress=40,TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=4} },Predecessor = "15FS" },
+
+                new TaskData() { TaskId = 17,TaskName = "Framing",StartDate = new DateTime(2019, 04, 04), EndDate = new DateTime(2019,04,28),Work=60,Progress=40,TaskType="FixedDuration" },
+
+                new TaskData() { TaskId = 18, TaskName = "Add load-bearing structure", StartDate = new DateTime(2019, 04, 03), Duration = "2",ParentId = 17,Work=60,Progress=20,TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=5} } },
+
+                new TaskData() { TaskId = 19, TaskName = "Natural gas utilities",StartDate = new DateTime(2019, 04, 08),Duration = "5", ParentId = 17, Work=60, Progress=40, TaskType="FixedDuration",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=5} },Predecessor = "18" },
+
+                new TaskData() { TaskId = 20, TaskName = "Electrical utilities", StartDate = new DateTime(2022, 04, 01),Duration = "4",ParentId = 17,Work=60,Progress=50,TaskType="FixedWork",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=5} },Predecessor = "19FS + 1" },
+
+                new TaskData() { TaskId = 21, TaskName = "Plumbing test", StartDate = new DateTime(2019, 04, 04), Duration = "4", Work=60, Progress=50, TaskType="FixedWork",Resources = new List<ResourceAlloacteData>(){ new ResourceAlloacteData() { ResourceId=6} } },
```

Comment:
Give only minimal records required as we in documentation pager we have only limited space for viewing

Reply:
updated